source/android/Renaissance/lib/add_resource_page.dart

```diff
@@ -0,0 +1,87 @@
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:portakal/my_colors.dart';
+
+class AddResourcePage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _AddResourcePageState();
+
+}
+
+class _AddResourcePageState extends State<AddResourcePage> {
+  TextEditingController descriptionController = TextEditingController();
+  TextEditingController linkController = TextEditingController();
```

Comment:
I think using local scoped variables via underscore ( like _descriptionController) is a better approach for these controllers